pkgs/by-name/he/headplane-agent/package.nix

```
@@ -0,0 +1,32 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+}:
+let
+  version = "0.5.10";
+in
+buildGoModule {
+  pname = "hp_agent";
+  src = fetchFromGitHub {
+    repo = "headplane";
+    owner = "tale";
+    rev = version;
```

using finalAttrs will make is so overriding the version will also apply to the tag
```suggestion
buildGoModule (finalAttrs: {
  pname = "hp_agent";
  version = "0.5.10";
  
  src = fetchFromGitHub {
    repo = "headplane";
    owner = "tale";
    tag = finalAttrs.version;
```

---

@awwpotato Thank you for the feedback!

I think it's going to fail with something like `expected a set, got a lambda function`.

Is there any way to run CI locally? I would like to avoid pushing only to see if it passes the checks.

---

Yep: https://github.com/NixOS/nixpkgs/actions/runs/14474373880/job/40596271592?pr=398667